assembl/static2/tests/unit/utils/timeline.spec.js

```diff
@@ -19,4 +19,30 @@ describe('getCurrentPhaseIdentifier timeline behavior', () => {
     const identifier = getCurrentPhaseIdentifier(timeline);
     expect(identifier).toEqual('bar');
   });
+
+  it('should return that the debate is not started', () => {
+    const timeline = [];
+    const isDebateStarted = getIsDebateStarted(timeline);
+    expect(isDebateStarted).toEqual(false);
+  });
+
+  it('should return that the debate is not started', () => {
```

Comment:
`it('should return that the debate has not started if all phases have not started',`